src/components/SearchForm/SearchForm.jsx

```diff
@@ -1,7 +1,24 @@
 import React from "react";
 import "./SearchForm.css";
+import { useStore } from "../../store";
+import { useState } from "react";
 
 export default function SearchForm() {
+    const [search, setKeyword] = useState("");
+    const { getSavedCards } = useStore().currentUser;
+    const { getCards } = useStore().newsCards;
+
+    const handleChange = (e) => {
```

Comment:
**Might be better**

You can make a custom hook for inputs handling:

```js
export function useForm(inputValues) {
  const [values, setValues] = useState(inputValues);

  const handleChange = (event) => {
    const {value, name} = event.target;
    setValues({...values, [name]: value});
  };
  return {values, handleChange, setValues};
}

```

This way you can easily handle any form in your project

And now only one line of code enables form handling:

```jsx

const {values, handleChange, setValues} = useForm({})


```